nuxeo-arender-rendition-apb/tasks/default.yml

```diff
@@ -36,6 +36,15 @@
       file: arender_document_service_broker_logback.xml
 
 
+- name: "Set Pre Stop Scripts config {{ state }}"
+  k8s:
+    state: "{{ state }}"
+    definition: "{{ lookup('template', 'arender_pre_stop_scripts_config.yml.j2') | from_yaml }}"
+  loop:
+    - name: "{{ arender_pre_stop_scripts_configmap_name }}"
+      file: pre-stop.sh
+
+
```

Comment:
The idea of the statement below this one is to regroup all simple resource creation. 

I would add 

```
   - name: arender_pre_stop_scripts_config.yml.j2
     filename: pre-stop.sh
 ```

just before the `*_dc` items, change the content of the template to:

```
apiVersion: v1
kind: ConfigMap
metadata:
  labels:
    app: {{ name }}
    apb_name: '{{ apb_name }}'
    apb_plan_id: '{{ _apb_plan_id }}'
    apb_service_class_id: '{{ _apb_service_class_id }}'
    apb_service_instance_id: '{{ _apb_service_instance_id }}'
  name: arender_pre_stop_scripts_config_name
  namespace: {{ namespace }}
data:
  pre-stop.sh: |
    {{ lookup('file', item.filename) | indent(4) }}

```

The ability to configure the filename if for me not worth it, so it can even be suppressed.

And finally, remove this whole `k8s` statement.